build.cake

```diff
@@ -130,6 +132,68 @@ void DoArchive(string runtime, DirectoryPath inputFolder, FilePath outputFile)
 }
 
 /// <summary>
+///  Generate the scripts which target the OmniSharp binaries.
+/// </summary>
+/// <param name="outputRoot">The root folder where the publised (or installed) binaries are located</param>
+void CreateRunScript(string outputRoot, string scriptFolder)
```

Comment:
i think we put too many logic in the cake file, which make it gradually become unmanageable. we have limited intellisense for cake script. can we build this logic in a separate app or script or task?